app/src/main/res/values/strings.xml

```diff
@@ -444,6 +444,8 @@
     <string name="collection_open_tabs">Open tabs</string>
     <!-- Text for the menu button to remove a top site -->
     <string name="remove_top_site">Remove</string>
+    <!-- Postfix for private WebApp titles -->
+    <string name="pwa_site_controls_title_private">%1$s (Private Mode)</string>
```

Comment:
Can we add a comment above this that explains what the placeholder is here? Probably just "Placeholder is replaced with app name"